### PR TITLE
Replace Holy Father with High Priest

### LINKED ISSUE #823: The Holy Father
**Mod Version**
d22eaa51

**Please explain your issue in as much detail as possible:**
> Tyrande

> The Holy Father

**Upload screenshots of the problem localization:**

<details>
<summary>Click to expand</summary>

image

</details>

### PR DESCRIPTION
**Pull request by DJScias**
Hello there,

This would change a few of the occurrences where the game uses "Holy Father" to "High Priest" (title of Elune leader).

~There are, however, a few lines I've yet to change but I am unsure what route I should go with them:~
```
EVTDESC39241;An emissary from [From.GetTitledFirstName] has arrived in our court. According to the emissary, the Holy Father is concerned about our employment of heathens as advisors.  The Pope is asking of me to remove them from position.;Un émissaire [From.Get_du_TitledFirstName] est arrivé à la Cour. Selon lui, le Saint-Père n'apprécie pas que nous ayons des païens comme conseillers. Le Pape me demande de les démettre de leurs fonctions.;Ein Gesandter von [From.GetTitledFirstName] ist am Hofe angekommen. Laut dem Gesandten ist der Heilige Vater wegen der Beschäftigung von Heiden als Berater besorgt. Der Papst erwartet von mir, dass ich sie von ihrer Position entferne.;;Un emisario de [From.GetTitledFirstName] ha llegado a nuestra corte. Según el emisario, el Santo Padre está preocupado porque empleamos a paganos como asesores. El Papa me solicita que los destituya de sus cargos.;;;;;;;;;x
```
~Should this just get a change to High Priest and the rest can be the same? I'm not sure if the Elune worshippers would call others "heathens".~

```
EVTDESC39271;The Holy Father is most concerned about the loyalties of the bishops of our realm. [From.GetBestName] is asking for us to replace one of them with a bishop.;La loyauté des évêques de notre Royaume préoccupe le Saint-Père. [From.Get_only_Le_Title][From.GetBestName] nous demande de remplacer l'un d'eux par un autre évêque.;Der Heilige Vater ist sehr besorgt bezüglich der Loyalität der Bischöfe in unserem Reich. [From.GetBestName] ersucht uns darum, einen von ihnen durch einen Bischof zu ersetzen;;El Santo Padre está muy preocupado por la lealtad de los obispos de nuestros dominios. [From.GetBestName] nos solicita que sustituyamos uno de ellos por un obispo;;;;;;;;;x
```
~Pretty sure Elune doesn't have bishops, I'm guessing we'd call this priestesses or what would it be?~

```
EVTDESC39277;The Holy Father has granted you the [From.PrimaryTitle.GetFullName]. We will serve you faithfully as a vassal, and guide your rule in a Christian way.;Le Saint-Père vous a accordé [From.PrimaryTitle.Get_le_RealmFull]. Nous serons vos fidèles vassaux et nous guiderons votre règne dans le chemin de la chrétienté.;Der Heilige Vater hat uns die [From.PrimaryTitle.GetFullName] erteilt! Wir werden Euch als Vasall treu ergeben dienen und Eure Regentschaft auf christliche Art und Weise vertreten.;;El Santo Padre os ha concedido [From.PrimaryTitle.GetFullName]. Os serviremos religiosamente como vasallos y guiaremos vuestro mandato de forma cristiana.;;;;;;;;;x
```
~It says Christian, which isn't the case for the mod. Plus I'm not sure if it'd really be called "vassal"? I don't know.~

**No longer WIP as the above localization shouldn't be changed, it is for Christian- and Fraticelli-only.**

~Beeping the Team for information regarding those three blocks. Once I fix those occurrences this PR should be complete~.

PR complete and should fix #823.

### CODE REVIEW

**Comment by geffe1** [comment]
Shouldn't it be "High Priestess"?

**Comment by zumbak04** [comment]
@DJScias do you want to replace all mentions of the Holy Father even if they are not used in the mod?

**Comment by DJScias** [comment]
@geffe1 I brought this up here: https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/842#issue-327741584

The religious title is just called "High Priest", so I went with that.

**Comment by DJScias** [comment]
Well, that depends if the user can request excommunication and try to revoke excommunication? If that part is disabled, then the extra lines aren't necessary I guess. @zumbak04

**Comment by zumbak04** [comment]
> Well, that depends if the user can request excommunication and try to revoke excommunication? If that part is disabled, then the extra lines aren't necessary I guess. @zumbak04

I mostly mean text from your first message.
Like `EVTDESC39241` is about...

> Pope asks a ruler that they should not have heathens employed.

`EVTDESC39271` is about...

> You have a bishop as your vassal, and the the vassal likes you more than the pope.

`EVTDESC39277` is about reshuffling of bishops and can fire for the Pope only.

**Comment by DJScias** [comment]
Right, so do you think the ones I have localized now are enough?

File: localisation/00_zuwc_united_localisation.csv [hunk 1]
```diff
@@ -2885,10 +2885,10 @@ EVTOPT_MNM_4018;Cognition is what I live for.;;Erkenntnis ist das, wof
...
+indulgence_ask;The High Priest will hear our plea for salvation.;Le Grand Pr�tre entendra notre plaidoyer pour le salut.;Der Hohepriester wird unsere Bitte um Erl�sung erh�ren.;;El Sumo Sacerdote oir� nuestra petici�n de salvaci�n.;;;;;;;;;x
+repentance_ask;The High Priest will hear our plea for salvation.;Le Grand Pr�tre �coutera notre demande de pardon.;Der Hohepriester wird unser Flehen um Erl�sung anh�ren.;;El Sumo Sacerdote escuchar� nuestra oraci�n de salvaci�n.;;;;;;;;;x
+EVTDESC39253;The High Priest has agreed to lift the excommunication against me!;Le Grand Pr�tre a accept� de lever mon excommunication!;Der Hohepriester hat zugestimmt, meine Exkommunikation aufzuheben!;;�El Sumo Sacerdote ha accedido a revocar la excomuni�n contra mi!;;;;;;;;;x
+EVTDESC39276;The High Priest wishes to inform us that we will now also obey an early liege.;Le Grand Pr�tre souhaite nous annoncer que nous ob�irons aussi � un jeune Seigneur.;Der Hohepriester w�nscht, uns dar�ber in Kenntnis zu setzen, dass wir uns nun auch einem fr�heren Lehnsherren unterwerfen werden m�ssen.;;El Sumo Sacerdote desea informarnos de que ahora tambi�n deberemos obedecer a un se�or terrenal.;;;;;;;;;x
```
Comment (zumbak04) [inline localisation/00_zuwc_united_localisation.csv]: This is from catholic and fraticelli only event chain.

**Comment by zumbak04** [comment]
> Right, so do you think the ones I have localized now are enough?

Yep, except the loc I mentioned above. Plus, I would make dynamic localization of religious head `[Root.RelHead.GetTitle]`.

**Comment by DJScias** [comment]
Pushed new version, removed that one localization and made the religious head title dynamic.

**Comment by zumbak04** [comment]
> Pushed new version, removed that one localization and made the religious head title dynamic.

Waiting for it to stop being WIP.

**Comment by DJScias** [comment]
Updated the title and main post. This should no longer be a WIP as the issues with those three localization blocks were deemed unnecessary to the mod.

File: localisation/00_zuwc_united_localisation.csv [hunk 2]
```diff
@@ -2885,10 +2885,10 @@ EVTOPT_MNM_4018;Cognition is what I live for.;;Erkenntnis ist das, wof
...
+indulgence_ask;The [Root.RelHead.GetTitle] will hear our plea for salvation.;Le [Root.RelHead.GetTitle] entendra notre plaidoyer pour le salut.;Der [Root.RelHead.GetTitle] wird unsere Bitte um Erl�sung erh�ren.;;El [Root.RelHead.GetTitle] oir� nuestra petici�n de salvaci�n.;;;;;;;;;x
+repentance_ask;The [Root.RelHead.GetTitle] will hear our plea for salvation.;Le [Root.RelHead.GetTitle] �coutera notre demande de pardon.;Der [Root.RelHead.GetTitle] wird unser Flehen um Erl�sung anh�ren.;;El [Root.RelHead.GetTitle] escuchar� nuestra oraci�n de salvaci�n.;;;;;;;;;x
+EVTDESC39253;The [Root.RelHead.GetTitle] has agreed to lift the excommunication against me!;Le [Root.RelHead.GetTitle] a accept� de lever mon excommunication!;Der [Root.RelHead.GetTitle] hat zugestimmt, meine Exkommunikation aufzuheben!;;�El [Root.RelHead.GetTitle] ha accedido a revocar la excomuni�n contra mi!;;;;;;;;;x
```
Comment (zumbak04) [inline localisation/00_zuwc_united_localisation.csv]: Looks like this localization isn't used anymore in the vanilla. These event not called anywhere. I would delete it.

Comment (zumbak04) [reply localisation/00_zuwc_united_localisation.csv]: Meh, no, fine, keep it. Who knows? Maybe it's used somewhere.

Comment (DJScias) [reply localisation/00_zuwc_united_localisation.csv]: Alright, I'll keep it.

**Comment by zumbak04** [comment]
@DJScias merge?

**Comment by DJScias** [comment]
Should be good to merge.

**Pull request merged**